File: day007/day007.py
import time

print("Girdiğiniz sayının rakamlarının karesi toplayan programa hoş geldiniz. ")
time.sleep(2)

sum = 0
x = int(input("Bir sayı giriniz: "))

sum += (x % 10) ** 2 
sum += int((x / 10)) ** 2

print(f"Girdiğiniz sayının rakamlarının toplamı {sum}")

File: day007/day007.cpp
#include <iostream>

using namespace std;

int main()
{
    int x, sum = 0;
    
    cout<<"Bir sayı giriniz. Bu sayının rakamlarının karesi size söylenecektir. ";
    cin >> x;
    sum += (x % 10) * (x % 10); 
    sum += (x / 10) * (x / 10);
    cout << sum;
    return 0;
}
